package.json

```diff
@@ -48,9 +49,9 @@
     "@babel/core": "^7.6.2",
     "@babel/runtime": "^7.6.2",
     "@react-native-community/eslint-config": "^0.0.5",
-    "@react-native-community/cli": "4.6.3",
-    "@react-native-community/cli-platform-ios": "4.6.3",
-    "@react-native-community/cli-platform-android": "4.6.3",
+    "@react-native-community/cli": "^3.1.0",
```

Comment:
Are there issues with the the newer version of RNCLI?

Reply:
4.x has a peer dependency on RN 62, the test apps are on 61 so it generated bunch of warnings in `yarn install`. 3.x is probably a better match for RN 61.